docs/source/markdown/podman-image-sign.1.md

```diff
@@ -41,6 +45,8 @@ Sign the busybox image with the identity of foo@bar.com with a user's keyring an
 
    sudo podman image sign --sign-by foo@bar.com --directory /tmp/signatures docker://privateregistry.example.com/foobar
 
+   sudo podman image sign --sigfile=/tmp/foobar.sig --sign-by foo@bar.com --directory /tmp/signatures docker://privateregistry.example.com/foobar
```

Comment:
I can’t see how it makes sense to use `--directory` and `--sigfile` together.

---

I could even see an argument that using the two together should explicitly fail.